portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,24 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random piece of information to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function addRandomInformation() {
+    const informations =
+        ['I like cats!', 'I was born in Tianjin, China!', 'I play badminton!', 'I like philosophy!', 'I grew up in the Greater Toronto Area!'];
+    const images = 
+        ["images/oat.jpg", "images/tianjin.png", "images/badminton.png", "images/philosophy.avif", "images/gta.webp"];
```

Comment:
This approach works for now.  If in the future you added more fields (like maybe a title, a paragraph, and an image) it would likely be easier to have 1 array like so:

```
const informations = [
    {
        title: 'I like cats!',
        body: 'Cats are great, let me tell you about....',
        image: 'images/oat.jpg'
    },
    {
        tiitle: 'I was born in Tianjin, China!',
        image: 'images/tianjin.png'
    },
    ...
]
```